functions/public/styles/index.css

```diff
@@ -133,43 +69,7 @@ body{
     border-radius: 0px;
     cursor: pointer;
 }
-#login-button-google{
```

Comment:
Google and Twitter sign in buttons not needed and therefore removed, including footer styling as it isn't required